dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.6/com.ibm.websphere.appserver.org.eclipse.persistence-2.6.feature

```diff
@@ -0,0 +1,12 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.6
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink; apiJar=false; location:=dev/api/third-party/, \
+  com.ibm.ws.javaee.persistence.2.1; location:=lib/
+kind=beta
```

Comment:
this will need to be `kind=ga`, since this feature will be included in GA'd features such as jpa-2.1